.github/workflows/lint-python.yml

```diff
@@ -6,15 +6,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
-      - run: pip install -r requirements_build.txt
-      - run: bandit -c pyproject.toml -r .
+      - run: pip install .[build]
```

Comment:
Does this work?
```suggestion
      - run: pip install .[build,text]
```

Reply:
Not sure, but I thought you wanted to segregate build and test deps in your GHA workflows? IMHO it's marginally easier to debug any GHA fails if you have the separate steps.